src/index.ts

```diff
@@ -1240,13 +1241,15 @@ export type TokenState = {
 
 export const POS_NEG_STORAGE_KEY = "sajari_tokens";
 // Just here to handle SSR execution (docs)
-const setItem = (key: string, value: string) => {
+export const setItem = (key: string, value: string) => {
   if (isSSR()) {
     return;
   }
-  return localStorage.setItem(key, value);
+  try {
+    localStorage.setItem(key, value);
+  } catch (_) {}
```

Comment:
Let's check if console log (or console error) exists and if so, log something meaningful to it. Unlikely, but it's possible that a customer discovers this erroring out and reports it to us.

Reply:
Good idea. 👍